app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<nav class="navbars">
```

Comment:
- Kindly follow the design guidelines for the Signup page:
   - [ ] Please add the `Next link` in the navbar in the header responsible for the sign-up process instead of adding a signup button.
   - [ ] Kindly decrease the top and bottom spacing for the header.

| Your design | Guidelines |
| --- | --- |
|![image](https://user-images.githubusercontent.com/24703171/230421953-71373176-f530-4235-b7d2-69d3522f898f.png) | ![image](https://user-images.githubusercontent.com/24703171/219884879-31b72a01-5fe0-4116-8fee-3a397873ebb9.png) |